plotters/src/element/pie.rs

```diff
@@ -163,8 +189,13 @@ impl<'a, DB: DrawingBackend, Label: Display> Drawable<DB> for Pie<'a, (i32, i32)
                 let label_size = backend.estimate_text_size(&perc_label, percentage_style)?;
                 let text_x_mid = (label_size.0 as f64 / 2.0).round() as i32;
                 let text_y_mid = (label_size.1 as f64 / 2.0).round() as i32;
+                let perc_radius = if self.donut_hole == &0.0 {
+                    self.radius / 2.0
+                } else {
+                    (self.radius + self.donut_hole) / 2.0
```

Comment:
I think the equation is wrong. 
If written like this, you don't even need the new the if statement because if `self.donut_hole == &0.0` is true, `(self.radius + self.donut_hole) / 2.0` is identical to `self.radius / 2.0`.

I think the correct equation for the donut case would be `self.radius + self.donut_hole / 2.0`.

Reply:
The correct equation is that originally written by me. We need the average between the external radius and the internal one.
But as you noticed, actually there is no need of the if statement.

Reply:
Fixed in https://github.com/plotters-rs/plotters/pull/485/commits/8f39a5eda0fee945c24810f10d0cecb6e7055fd6